node/pkg/adminrpc/adminserver.go

```diff
@@ -1003,3 +1007,95 @@ func (s *nodePrivilegedService) DumpRPCs(ctx context.Context, req *nodev1.DumpRP
 		Response: s.rpcMap,
 	}, nil
 }
+
+func (s *nodePrivilegedService) GetAndObserveMissingVAAs(ctx context.Context, req *nodev1.GetAndObserveMissingVAAsRequest) (*nodev1.GetAndObserveMissingVAAsResponse, error) {
+	// Get URL and API key from the command line
+	url := req.GetUrl()
+	apiKey := req.GetApiKey()
+
+	// Create the body of the request
+	jsonBody := []byte(`{"apiKey": "` + apiKey + `"}`)
+	jsonBodyReader := bytes.NewReader(jsonBody)
+
+	// Create the actual request
+	httpRequest, err := http.NewRequestWithContext(ctx, http.MethodPost, url, jsonBodyReader)
+	if err != nil {
+		fmt.Printf("client: could not create request: %s\n", err)
+		os.Exit(1)
+	}
+
+	httpRequest.Header.Set("Content-Type", "application/json")
+
+	client := http.Client{
+		Timeout: 30 * time.Second,
+	}
+
+	// Call the cloud function to get the missing VAAs
+	results, err := client.Do(httpRequest)
+	if err != nil {
+		fmt.Printf("GetMissingVAAs: error making http request: %s\n", err)
+		return nil, err
+	}
+
+	// Collect the results
+	resBody, err := io.ReadAll(results.Body)
+	if err != nil {
+		fmt.Printf("GetMissingVAAs: could not read response body: %s\n", err)
+		return nil, err
+	}
+	fmt.Printf("client: response body: %s\n", resBody)
+	type MissingVAA struct {
+		chain  int
+		txhash string
+		vaaKey string // "<chain>/<emitter>/<sequence>"
+	}
+	var missingVAAs []MissingVAA
+	err = json.Unmarshal(resBody, &missingVAAs)
+	if err != nil {
+		fmt.Printf("GetMissingVAAs: could not unmarshal response body: %s\n", err)
+		return nil, err
+	}
+
+	// Only do a max of 25 at a time so as to not overload the node
+	processingLen := len(missingVAAs)
+	if processingLen > 25 {
+		processingLen = 25
+	}
+
+	// Start injecting the VAAs
+	obsCounter := 0
+	for i := 0; i < processingLen; i++ {
+		missingVAA := missingVAAs[i]
+		// First check to see if this VAA has already been signed
+		// Convert vaaKey to VAAID
+		splits := strings.Split(missingVAA.vaaKey, "|")
+		chainID, err := strconv.Atoi(splits[0])
+		if err != nil {
+			continue
+		}
+		sequence, err := strconv.ParseUint(splits[2], 10, 64)
+		if err != nil {
+			continue
+		}
+		vaaKey := db.VAAID{EmitterChain: vaa.ChainID(chainID), EmitterAddress: vaa.Address([]byte(splits[1])), Sequence: sequence}
+		hasVaa, err := s.db.HasVAA(vaaKey)
+		if err != nil || hasVaa {
+			continue
+		}
+		var obsvReq gossipv1.ObservationRequest
+		obsvReq.ChainId = uint32(missingVAA.chain)
+		obsvReq.TxHash = []byte(missingVAA.txhash)
+		// Call the following function to send the observation request
+		if err := common.PostObservationRequest(s.obsvReqSendC, &obsvReq); err != nil {
+			continue
```

Comment:
Log an error here? Maybe return something to the user?

Reply:
Do you think this change is sufficient?  Do you think I should have a running error msg and put that into the response?  Or, just fmt.Println() the errors?

Reply:
If someone wants to script this command, would it be better to have something they could easily grep for in the output to tell them it wasn't 100% successful?

Reply:
I added the error counter.  We could start with that for now.  I can add more later, if people ask for it.